README.md

```diff
@@ -105,6 +105,17 @@ You can provide an optional list of custom JS files, which must be placed inside
 
 See the [example site config file](https://github.com/zwbetz-gh/cupper-hugo-theme/blob/master/exampleSite/config.yaml) for sample usage.
 
+## Default to Dark Theme
```

Comment:
Add this heading to the Table of Contents

Reply:
Also, add a note to this section, that local storage must be cleared in order for this param to take effect.

Reply:
So far, I've got the following text. Please let me know of any edits you'd like.

----

Note that the default of light or dark theme only applies to the first visit to a site using this theme. Once the site is visited the choice of dark or light theme is stored in 'local storage' in the browser.

To reset to a 'first visit' scenario (e.g. for testing), one needs to either browse in private mode (aka Incognito/InPrivate/etc) or delete 'local storage' for this site. The easiest way to do that, but which affects other sites as well, is to use the 'Clear History' feature of the browser.

Check your browser's help or documentation for details.